include/dnscpp/tcp.h

```diff
@@ -103,6 +106,18 @@ class Tcp : public Socket, private Monitor, private Connecting
      */
     bool _connected = false;
 
+    /**
+     *  Query IDs currently sent over the wire
+     *  @var std::set
+     */
+    std::set<uint16_t> _queryids;
+
+    /**
+     *  Queries awaiting to be sent over wire, but had an ID collision with one that is already in flight
+     *  @var std::map
```

Comment:
std::multimap<uint16_t,Query> is easier to read. Add in-code comment to explain why we did not use that.

Reply:
agreed

Reply:
fixed in 0e95472